tools/platyPSHelp/help/platyPSHelp.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This is the module page that is generated by platyPS when the markdown files are initially created. It is used when `Update-MarkdownHelpModule` is executed.